nilearn/input_data/nifti_labels_masker.py

```diff
@@ -57,8 +57,17 @@ class NiftiLabelsMasker(BaseMasker, CacheMixin):
         millimeters of the spatial smoothing to apply to the signal.
 
     standardize: boolean, optional
-        If standardize is True, the time-series are centered and normed:
-        their mean is put to 0 and their variance to 1 in the time dimension.
+        If standardize is True, the time-series are normalized using
```

Comment:
Same comment here on adding more details about referring standardize_strategy to look for strategies to standardize the signal.